great_tables/data/__init__.py

```diff
@@ -1,9 +1,17 @@
-import pandas as pd
-import pkg_resources
+from importlib_resources import files
 
-DATA_MOD = "great_tables.data"
+try:
```

Comment:
Give a friendly message when users don't have pandas, so raise an error trying to import data.